feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/BookmarkList.kt

```diff
@@ -36,65 +45,90 @@ fun BookmarkList(
     onBookmarkIconClick: (TimetableItem) -> Unit,
     modifier: Modifier = Modifier,
 ) {
+    val density = LocalDensity.current
     LazyColumn(
         state = scrollState,
         modifier = modifier.padding(end = 16.dp),
     ) {
-        timetableItemMap.forEach { (_, values) ->
-            itemsIndexed(values) { index, timetableItem ->
-                Row(modifier = Modifier.padding(top = 10.dp)) {
+        itemsIndexed(timetableItemMap.toList(), key = { _, (key, _) -> key }) { index, (_, values) ->
```

Comment:
We might have to consider that the item for LazyColumn is going to be a time range.
So even if we don't see all the items for the time range, it will be created.
On the other hand, if we see the scroll state and make other LazyColumn for time, we don't have this issue.
I'm not sure if this problem is noticeable 🤔

Reply:
Um..., the number of timeTable item which an item of LazyColumn is 4 or five at most. Size of 4 item is the same as that of Screen. I think it is not a big problem.
The size of item Of LazyColumn are different from each other, so it is difficult to calclate those each height and apply Offset to TimeText.

Reply:
Yeah, it works awesome and codes are great so if we need to improve I think we can. I'm willing to merge this.

Reply:
I'll mark this as solved.